kkomaz/debut
# Add users from contacts

This PR 
* adds a call to add all contacts with a blockstack id to the list of following users.

I didn't manage to make the final way through the tests because the lookup of the other users failed in my local development environment

Steps for testing:
1. follow the steps in https://docs.blockstack.org/develop/collections.html (ignore that you need to use a custom gaia storage, collections have been deployed to the default gaia by Blockstack PBC)
1. Add some users in your address book
1. Refresh Debut app and see that the users have been added to the list of following
1. enjoy the moments by your close ones

This is kind of cool because you can share your contacts without worrying what the social network is doing with your contact datas. It all stays savely with you and you can verify it because debut app is open source, so you can run your own.
